### PR TITLE
Add collect_by_instance, collect_without_dimension options to cloudwatch handler

### PR DESCRIPTION
**Pull request by bandesz**
This change will allow us not to only collect metrics for the instances, but without any dimension.

### CODE REVIEW

File: src/diamond/handler/cloudwatch.py [hunk 1]
```diff
@@ -25,13 +25,17 @@
 namespace = MachineLoad
 name = Avg01
 unit = None
+collect_by_instance = True
+collect_without_dimension = False
```
Comment (shortdudey123) [inline src/diamond/handler/cloudwatch.py]: Alphabetize

File: src/diamond/handler/cloudwatch.py [hunk 2]
```diff
@@ -25,13 +25,17 @@
...
 
 [[[LoadAvg05]]]
 collector = loadavg
 metric = 05
 namespace = MachineLoad
 name = Avg05
 unit = None
+collect_by_instance = True
+collect_without_dimension = False
```
Comment (shortdudey123) [inline src/diamond/handler/cloudwatch.py]: Alphabetize

File: docs/handlers/cloudwatchHandler.md [hunk 3]
```diff
@@ -29,22 +29,26 @@ metric = 01
...
 collector | loadavg |  | str
-get_default_config_help |  | get_default_config_help | 
 metric | 01 |  | str
 name | Avg01 |  | str
 namespace | MachineLoad |  | str
-region | us-east-1 |  | str
-server_error_interval | 120 | How frequently to send repeated server errors | int
 unit | None |  | str
+collect_by_instance | True | Collect metrics for instances separately | bool
+collect_without_dimension | False | Collect metrics without dimension | bool
```
Comment (shortdudey123) [inline docs/handlers/cloudwatchHandler.md]: Alphabetize

File: src/diamond/handler/cloudwatch.py (hunk 1 above)
Comment (bandesz) [inline src/diamond/handler/cloudwatch.py]: fixed

File: src/diamond/handler/cloudwatch.py (hunk 2 above)
Comment (bandesz) [inline src/diamond/handler/cloudwatch.py]: fixed

File: docs/handlers/cloudwatchHandler.md (hunk 3 above)
Comment (bandesz) [inline docs/handlers/cloudwatchHandler.md]: fixed

**Comment by shortdudey123** [comment]
👍  LGTM
@jaingaurav @bmhatfield review?

**Pull request merged**